<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab_survey</class>
 <widget class="QWidget" name="tab_survey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>802</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="checkBox_topo">
       <property name="text">
        <string>Load topography data</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_topo"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_topoBrowser">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="src_setting">
       <attribute name="title">
        <string>Source setting</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_src" stretch="0,0,1,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_topo_src" stretch=""/>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_srcnum">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_srcint">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Electrodes number: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_srcint">
            <property name="text">
             <string>Discrete interval (m): </string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_src_table">
          <property name="leftMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_clear_src">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_import_src">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_save_src">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rcv_setting">
       <attribute name="title">
        <string>Receiver setting</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_rcv" stretch="0,0,1,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_topo_rcv" stretch=""/>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="2">
           <widget class="QLabel" name="label_min">
            <property name="font">
             <font>
              <family>Times</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>End (m)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_rcvXmax">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEdit_rcvYmax">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_rcvXmin">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_X">
            <property name="font">
             <font>
              <family>Times</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>X space: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_int">
            <property name="font">
             <font>
              <family>Times</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Interval (m)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="lineEdit_rcvYint">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_Y">
            <property name="font">
             <font>
              <family>Times</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Y space: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEdit_rcvXint">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_max">
            <property name="font">
             <font>
              <family>Times</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Start (m)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_rcvYmin">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_rcv_table">
          <property name="leftMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0,0">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_clear_rcv">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_import_rcv">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_save_rcv">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,1,0,0">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>  src config: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_srcfile">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>rcv config: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_rcvfile">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_view">
           <property name="text">
            <string>View Survey</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_viewSurvey">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>View Survey</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_view"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_srcnum</tabstop>
  <tabstop>lineEdit_srcint</tabstop>
  <tabstop>pushButton_save_src</tabstop>
  <tabstop>lineEdit_rcvXmin</tabstop>
  <tabstop>lineEdit_rcvXmax</tabstop>
  <tabstop>lineEdit_rcvXint</tabstop>
  <tabstop>lineEdit_rcvYmin</tabstop>
  <tabstop>lineEdit_rcvYmax</tabstop>
  <tabstop>lineEdit_rcvYint</tabstop>
  <tabstop>pushButton_save_rcv</tabstop>
  <tabstop>pushButton_view</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
